lasagne/nonlinearities.py

```diff
@@ -22,10 +34,30 @@ def rectify(x):
 
 
 # leaky rectify
-# Maas et al: Rectifier Nonlinearities Improve Neural Network Acoustic Models
-# http://web.stanford.edu/~awni/papers/relu_hybrid_icml2013_final.pdf
 class LeakyRectify(object):
+    """Implementation of a leaky rectifier.
```

Comment:
Since we have formulas for `rectify` and `identity`, it might be nice to add one here as well for consistency.